servo-media/src/audio/node.rs

```diff
@@ -92,6 +136,9 @@ pub enum AudioNodeMessage {
     AudioBufferSourceNode(AudioBufferSourceNodeMessage),
     GainNode(GainNodeMessage),
     OscillatorNode(OscillatorNodeMessage),
+    SetChannelCount(u8),
+    SetChannelMode(ChannelCountMode),
+    SetChannelInterpretation(ChannelInterpretation),
```

Comment:
Should these setters get a `Sender<()>` so we can block from the consumer (DOM side in this case) until the property is actually set?

Reply:
We don't block for the other setters. I don't see a reason to block the DOM thread on the rendering thread.